ZConfig/cmdline.py

```diff
@@ -155,7 +157,7 @@ def get_section_info(self, type, name):
             bk = self.basic_key(s, pos)
             if name and self._normalize_case(s) == name:
                 L.append((optpath[1:], val, pos))
-            elif bk == type:
+            elif bk == type: # pragma: no cover
```

Comment:
``type`` has been a built-in for a very long time (since before I discovered Python at version 1.2).  I've never been particularly averse to shadowing it, though.